scanner/src/main/java/edu/ucr/cs/riple/scanner/ErrorProneCLIFlagsConfig.java

```diff
@@ -37,7 +39,7 @@
 
 public class ErrorProneCLIFlagsConfig implements Config {
 
-  public final Path outputDirectory;
+  @Nonnull public final Path outputDirectory;
```

Comment:
Shouldn't all these fields be private? Particularly if we are calling the methods from `Config` consistently to get this info through those.